backend/migrations/2023-07-12-143111_user_data/up.sql

```diff
@@ -0,0 +1,34 @@
+-- Your SQL goes here
+CREATE TYPE salutation AS ENUM (
+    'Ms',
+    'Mrs',
+    'Mr',
+    'Mx'
```

Comment:
Probably should we also make this lower-case to be consistent. Maybe male/female/other is better suited as 3 options?